runtime/test/manifest-test.js

```diff
@@ -475,26 +473,6 @@ describe('manifest', function() {
     let manifest = await Manifest.load('somewhere/a', loader, {registry});
     assert(registry['somewhere/a path/b']);
   });
-  it('parses all particles manifests', async () => {
```

Comment:
Where did this move or what made it not relevant any more?

Reply:
I was just being lazy, this was the only place in the codebase that imported 'path'. I've put it back now.